pkg/operator/api/api_test.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2016 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package api
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/rook/rook/pkg/model"
+	testop "github.com/rook/rook/pkg/operator/test"
+
+	"github.com/stretchr/testify/assert"
+	"k8s.io/client-go/kubernetes/fake"
+	"k8s.io/client-go/pkg/api/v1"
+)
+
+func TestStartAPI(t *testing.T) {
+	clientset := testop.New(3)
+	info := testop.CreateClusterInfo(1)
+	c := New("ns", "myversion")
+
+	// start a basic cluster
+	err := c.Start(clientset, info)
+	assert.Nil(t, err)
+
+	validateStart(t, c, clientset)
+
+	// starting again should be a no-op, but still results in an error
```

Comment:
comment says it results in an error but then we assert that err is nil.  Those are conflicting right?

i think this comment occurs in a couple other locations